src/api/entities.js

```diff
@@ -48,21 +47,14 @@ export const doLoadEndpoints = sourceList =>
     axiosInstance.get(`${SOURCES_API_BASE}/endpoints/?source_id=${sourceList}`);
 
 export function doRemoveSource(sourceId) {
-    return getSourcesApi().deleteSource(sourceId).then((sourceDataOut) => {
-        console.log('API call deleteSource returned data: ', sourceDataOut);
-    }, (_error) => {
-        console.error('Source removal failed.');
+    return getSourcesApi().deleteSource(sourceId).catch((_error) => {
```

Comment:
I think we should not be hiding the error. The notification component can handle that.

Reply:
Created :+1: